src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -319,7 +319,61 @@
   end
 
   describe '#user_and_role' do
-    skip('Please add some tests')
+    let(:user) { create(:user) }
+    let(:nobody) { create(:user_nobody) }
```

Comment:
The nobody user gets created in the seeds.rb, however, we remove the users with DatabaseCleaner
https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/support/database_cleaner.rb#L4